config/database.go

```diff
@@ -51,10 +59,10 @@ func DbURL(dbConfig *DBConfig) string {
 	)
 }
 
-func MysqlConn() (*gorm.DB, error) {
-	db, err := gorm.Open(mysql.Open(DbURL(BuildDBConfig())), &gorm.Config{})
+func (dbsql *DBMySQL) MysqlConn() *gorm.DB {
```

Comment:
I just created a new version of MySQL conn. This way will make our app is possible to run multiple data source at the same time (sql & no-sql). Models will easily integrated from multiple data source by calling the struct on their files.